File: src/main/java/com/wmstein/tourcount/database/Section.java
package com.wmstein.tourcount.database;

import java.text.DateFormat;
import java.text.SimpleDateFormat;
import java.util.Date;

/**
 * Created by milo on 05/05/2014.
 * Changed by wmstein on 18.02.2016
 */
public class Section
{
    public int id;
    public long created_at;
    public String name;
    public String country;
    public String plz;
    public String city;
    public String place;
    public int temp;
    public int wind;
    public int clouds;
    public String date;
    public String start_tm;
    public String end_tm;
    public String notes;

    //Get Date from DB table sections field created_at
    public String getDate()
    {
        Date date = new Date(created_at);
        DateFormat df = SimpleDateFormat.getDateInstance();
        return df.format(date);
    }

}

File: tourcount/src/main/java/com/wmstein/tourcount/database/Section.kt
package com.wmstein.tourcount.database

/**********************************
 * Definitions for table Section
 * Created by wmstein on 2016-02-18,
 * last edited in Java on 2022-03-23,
 * converted to Kotlin on 2023-07-05,
 * last edited on 2026-02-24
 */
class Section {
    @JvmField
    var id = 0
    @JvmField
    var name: String? = ""
    @JvmField
    var country: String? = ""
    @JvmField
    var plz: String? = ""
    @JvmField
    var city: String? = ""
    @JvmField
    var place: String? = ""
    @JvmField
    var tmp = 0
    @JvmField
    var wind = 0
    @JvmField
    var clouds = 0
    @JvmField
    var tmp_end = 0
    @JvmField
    var wind_end = 0
    @JvmField
    var clouds_end = 0
    @JvmField
    var date: String? = ""
    @JvmField
    var start_tm: String? = ""
    @JvmField
    var end_tm: String? = ""
    @JvmField
    var notes: String? = ""
    @JvmField
    var b_state: String? = ""
    @JvmField
    var st_locality: String? = ""
}
